src/classes/Client.ts

```diff
@@ -15,8 +15,13 @@ import AI from './AI';
 import { version } from '../../package.json';
 import PhishingHandler from './handlers/PhishingHandler';
 
+interface applicationCommands {
+    chatInput: Collection<string, ChatInputCommand>;
+    contextMenu: Collection<string, ContextMenuCommand>;
+}
```

Comment:
can you move this to /types/applicationCommand?

Reply:
idk